.circleci/config.yml

```diff
@@ -594,6 +620,32 @@ jobs:
       - checkout
       - run: DEPLOY=watcher_info sh .circleci/ci_deploy.sh
 
+  run_load_test_on_deployed_env:
```

Comment:
Right now this step is not being run in CI (which is good IMO), but is it intended?